packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -3275,6 +3276,7 @@ class Stack extends MultiChildRenderObjectWidget {
     this.alignment = AlignmentDirectional.topStart,
     this.textDirection,
     this.fit = StackFit.loose,
+    this.overflow = Overflow.clip,
```

Comment:
I believe you also want to add the deprecation notice to this param here.

Reply:
Done.